infra/feast-operator/config/samples/charts_v1alpha1_feastfeatureserver.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: charts.feast.dev/v1alpha1
+kind: FeastFeatureServer
+metadata:
+  name: feastfeatureserver-sample
+spec:
+  # Default values copied from helm-charts/feast-feature-server/values.yaml
+  affinity: {}
+  # base64 encoding of `sdk/python/feast/templates/local/feature_repo/feature_store.yaml`
+  feature_store_yaml_base64: "cHJvamVjdDogbXlfcHJvamVjdAojIEJ5IGRlZmF1bHQsIHRoZSByZWdpc3RyeSBpcyBhIGZpbGUgKGJ1dCBjYW4gYmUgdHVybmVkIGludG8gYSBtb3JlIHNjYWxhYmxlIFNRTC1iYWNrZWQgcmVnaXN0cnkpCnJlZ2lzdHJ5OiBkYXRhL3JlZ2lzdHJ5LmRiCiMgVGhlIHByb3ZpZGVyIHByaW1hcmlseSBzcGVjaWZpZXMgZGVmYXVsdCBvZmZsaW5lIC8gb25saW5lIHN0b3JlcyAmIHN0b3JpbmcgdGhlIHJlZ2lzdHJ5IGluIGEgZ2l2ZW4gY2xvdWQKcHJvdmlkZXI6IGxvY2FsCm9ubGluZV9zdG9yZToKICAgIHR5cGU6IHNxbGl0ZQogICAgcGF0aDogZGF0YS9vbmxpbmVfc3RvcmUuZGIKZW50aXR5X2tleV9zZXJpYWxpemF0aW9uX3ZlcnNpb246IDIK"
```

Comment:
shouldn't we document somewhere what's the actual feature_store.yaml that this is representing? (and maybe in the future we can also add a more complex sample with integrated psql DB deployment)

Reply:
it's currently documented in the comment above field... that said, i can certainly add something additional to the getting started docs i'll be adding.

Reply:
fwiw, to keep maintenance low... i'd like to lean heavily on existing feast docs and samples. there's no need/reason for us to duplicate anything. for example - 
https://github.com/feast-dev/feast/blob/master/sdk/python/feast/templates/postgres/feature_repo/feature_store.yaml
https://docs.feast.dev/project/development-guide#feast-python-feature-server-helm-chart

Reply:
i've added some additional docs, including how one might pass in their own `feature_store.yaml` during a deployment.